### PR TITLE
Add alternative to Netlify form for joining

### PR DESCRIPTION
**Pull request by anneke**

### CODE REVIEW

**Review by edwinm — APPROVED** [review]

File: src/en/join-us/index.md [hunk 1]
```diff
@@ -7,7 +7,12 @@ The contribution is {{ general-info.contribution }} (including 21% VAT). Members
...
+<div style="padding: var(--spacing-half) var(--spacing); background: #efefef;">
+
+**Please note:** Naturally, we will treat your data confidentially. Only the board will see this data. Your name and possible employer will be published on the public membership list, if desired. We do temporarily store form submissions with Netlify. If this is something you object to, you can also <a href="mailto:?subject=Membership%20Application&body=Name:%20%0A[FULL%20NAME]%0A%0AAddress:%20%0A[PERSONAL%20ADDRESS]%0A%0AEmail%20Address:%20%0A[EMAIL]%0A%0APhone%20Number:%20%0A[PHONE%20NUMBER]%0A%0AEmployer:%20%0A[EMPLOYER]%0A%0AStudent:%20%0A[YES/NO]%0A%0AVolunteer%20Interest:%20%0A[INTEREST%20IN%20VOLUNTEERING]%0A%0AHow%20Did%20You%20Learn%20About%20Fronteers:%20%0A[INTRODUCTION]%0A%0AExpectations%20of%20the%20Organization:%20%0A[EXPECTATIONS]%0A%0AAdditional%20Comments%20or%20Questions:%20%0A[COMMENTS]%0A%0AConsent%20for%20Member%20List%20Publication:%20%0A[YES/NO]%0A%0A">
```
Comment (sandervink) [inline src/en/join-us/index.md:12]: This link seems to be missing an actual e-mail address? :)

File: src/en/join-us/index.md [hunk 2]
```diff
@@ -7,7 +7,12 @@ The contribution is {{ general-info.contribution }} (including 21% VAT). Members
...
+send us an e-mail</a>. If this e-maillink does not work you can also copy this [e-mail template](https://github.com/fronteers/website/blob/main/templates/lidmaatschap.txt) in an e-mail.
```
Comment (sandervink) [inline src/en/join-us/index.md:13]: I think you've also made an English version of this file, right? If so, link to that?

File: src/en/join-us/index.md [hunk 3]
```diff
@@ -7,7 +7,12 @@ The contribution is {{ general-info.contribution }} (including 21% VAT). Members
 
 The contribution for students at a higher educational institution is {{ general-info.students_contribution }} (including VAT). Students who register after June 30 pay {{ general-info.students_discounted }}. 
 
-Naturally, we will treat your data confidentially. Only the board and the membership administration (appointed by the board) will see this data. Your name and possible employer will be published on the public membership list, if desired.
+<div style="padding: var(--spacing-half) var(--spacing); background: #efefef;">
```
Comment (sandervink) [inline src/en/join-us/index.md:10]: Is this newly introduced `div` (with a `<br />` after the closing tag) really necessary? Markdown should create its own spacing, right? (same remark for the Dutch version)

File: templates/lidmaatschap.txt [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+[EMAILADRES]
+
+Telefoonnummer:
+[TELEFOONNUMMER]
+
+Werkgever: we sturen de factuur niet naar je werkgever; je bent persoonlijk lid
+[WERKGEVER]
+
+Ja, ik ben student aan een hogere onderwijsinstelling: [JA/NEE]
+Om studentenkorting te ontvangen, dien je een bewijs van inschrijving van de hogere onderwijsinstelling waar je studeert aan het bestuur beschikbaar te maken. Stuur een scan mee!
```
Comment (sandervink) [inline templates/lidmaatschap.txt:19]: I think it would be a good idea to also add this description to the English version :)

**Pull request merged**